src/args/parsed_args.go

```diff
@@ -93,6 +93,9 @@ type ParsedArguments struct {
 	ConsumerGroupRegex *regexp.Regexp
 
 	Timeout int `default:"10000" help:"Timeout in milliseconds per single JMX query."`
+
+	// Zookeeper topic discovery
+	ZookeeperTopics bool
```

Comment:
For readability's sake I'd suggest to rename this to something more specific, e.g. `TopicSource` and having it default to `broker`. The other valid option would be `zookeeper`.

To me, current name suggest that you can _additionally_ collect "Zookeper topics", rather than it being a switch.